### ISSUE TITLE
Saving a response as example reverts changes done to the request

### ISSUE THREAD

**Issue by pmalecka** [a85/POSTMan-Chrome-Extension#10786]
### Is there an existing issue for this?

- [X] I have searched the tracker for existing similar issues

### Describe the Issue

When making a change to an existing request (e.g. body, path, headers, etc.) and then calling the request and saving the response as example - changes to the request are reverted.

### Steps To Reproduce

1. Open an existing request in a collection (created by duplicating another request)
2. Make a change to the request
3. Save the change
4. Send the request
5. Click Save Response -> Save as example
6. Observe the changes in the request reverted to their original value
7. Scratch your head a day later, as to why the changes to the request were not saved

### Environment Information

```markdown
- Operating System: Windows 10
- Platform Type: Native App
- Postman Version: Version 9.15.3
```

### Additional Context?

Shared workspace, team licence, synced collection. Reproducible 100%. Took us some time to figure out what's happening.

Applies to any change to the request.

Please escalate, this breaks our workflow.

**Comment by eugeniuszG** [comment]
Yeah
This is so frustrating!! Have the same issue. Noticed that this bug appears only with 200 response status code, with other statuses, when saving example, everything works just fine.

**Comment by eugeniuszG** [comment]
I can not create documentation because of this bug. Wtf guys!

**Comment by timbochamp** [comment]
Thanks for reporting this; I'm able to reproduce the same as well. I've logged a bug and we'll have our team investigate and report back here with a resolution

**Comment by giridharvc7** [comment]
@eugeniuszG @pmalecka Thanks for reporting this, We are looking into this right now. Will keep you posted. 

Can you confirm if this is just a UI Bug, meaning, if you are using the web, refreshing the browser tab fixes this and on the app, if you close and open it fixes this.

**Comment by pmalecka** [comment]
Seems to be just a UI bug, when I close postman and reopen (after step 6.), correct values are presented.

I got a silent update to 9.15.5, and I cannot replicate this anymore, did you release a fix already?

**Comment by timbochamp** [comment]
@pmalecka thanks for confirming. We were seeing the same after closing/reopening the app or the browser tab in Postman for Web.

Yes, the fix should be in place as of v9.15.5!

**Comment by pmalecka** [comment]
You guys are fast, thank you!
